src/redux/rockets/RocketSlice.js

```diff
@@ -0,0 +1,52 @@
+import { createSlice } from '@reduxjs/toolkit';
+/* eslint-disable */
```

Comment:
- [ ] It would be better to remove Eslint disable because Esline ensures that you write clean code